util/occamygen.py

```diff
@@ -404,10 +404,22 @@ def main():
     #######
     # DTS #
     #######
-    dts.add_chosen("stdout-path = \"{}\";".format(dts_uart))
+    # TODO(niwis, zarubaf): We probably need to think about genrating a couple
+    # of different systems here. I can at least think about two in that context:
+    # 1. RTL sim
+    # 2. FPGA
+    # 3. (ASIC) in the private wrapper repo
+    # I think we have all the necessary ingredients for this. What is missing is:
+    # - Create a second(/third) configuration file.
+    # - Generate the RTL into dedicated directories
+    # - (Manually) adapt the `Bender.yml` to include the appropriate files.
```

Comment:
Good idea. This would not only help for different DTSes but also cluster configurations (e.g. number of quadrants) for the emulation platform.

Reply:
Yes, exactly my thinking. I'll create an issue out of it :-)